docs/concepts/jobs-overview.md

```diff
@@ -0,0 +1,59 @@
+(jobs-overview)=
+# Jobs Overview
+
+A `Job` is simply something that you would like to run once to completion. For example, running a simple Python script can be similar to a `Job` in that the script runs once to completion and then returns. In an ML workflow, sometimes running a simple Python script is not enough and additional functionality is required. For example giving cluster resources, specifying dependencies and a Docker image, integrating with Git repo, etc. 
+
+### Create a Job
+
+MLRun has the ability to add all of the above features and more when running a `Job`. To showcase this, we will create a job to run the code below which resides in a file titled `code.py`:
```

Comment:
MLRun can add all of the above features, and more, when running a `job`. To showcase this, the following job runs the code below, which resides in a file titled `code.py`: